packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -49,20 +64,51 @@ function VariantRow({
         padding: '0px 24px',
         borderRadius: 8,
         border: variant.isBase ? '2px solid black' : '1px solid black',
+        position: 'relative',
       }}
+      role="group"
+      _hover={{
+        cursor: 'pointer',
+        backgroundColor: tinycolor(variant.color).isDark() ? 'white' : 'black',
```

Comment:
I actually found the fact that is hovers to a different color (white) in this instance a it confusing. Because it made me think the color was changing to white. Maybe increase its border to be 2px instead?

Reply:
Ah okay I tried reversing the colors. I had tried the border out but maybe ill make it bigger.